Cargo.toml

```diff
@@ -97,6 +97,23 @@ rustdoc-args = ["--cfg", "docsrs", "--cfg", "hyper_unstable_ffi", "--cfg", "hype
 [package.metadata.playground]
 features = ["full"]
 
+[package.metadata.cargo_check_external_types]
```

Comment:
Is this change required? Did they drop support for a separate file? I kinda like that this stuff is over in the CI folder, and not bothering the files that get published...

Reply:
It is not required. `cargo-check-external-types` supports a separate config file.

Reply:
Fixed.